stdlib/public/SwiftRemoteMirror/CMakeLists.txt

```diff
@@ -2,14 +2,35 @@
 # always built as a shared library.
 if(SWIFT_BUILD_DYNAMIC_STDLIB OR SWIFT_BUILD_REMOTE_MIRROR)
```

Comment:
I think SWIFT_BUILD_REMOTE_MIRROR needs to be removed from here and moved to the new host code. Not sure what this build setting is for. Is it always set at the same time as SWIFT_INCLUDE_TOOLS, or is it different?)

Reply:
It is up to the user to decide if the remote mirror should be built or not.  Personally, I am of the opinion that if Apple doesn't build in a configuration where it matters, we should remove the option and always build it when `SWIFT_INCLUDE_TOOLS` is defined.  We now have a differentiation between the host and target bits, and if we are building tools, we should just build it for the host.

Reply:
It's unclear to me whether you're agreeing with me that the SWIFT_BUILD_REMOTE_MIRROR should be moved to the host part, or whether you're arguing something different

Reply:
I'm arguing for something more drastic (the removal of the option).  I'm interested in how the current version doesn't work though as the `if(NOT SWIFT_BUILD_STDLIB AND NOT SWIFT_BUILD_REMOTE_MIRROR)` should prevent the duplicate umbrella target.